### PR TITLE
scoop-bucket, wsl, takeown: add Chinese translation

### PR DESCRIPTION
**Pull request by ChungZH**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh/windows/scoop-bucket.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# scoop bucket
+
+> Manage buckets: Git repositories containing files which describe how scoop installs applications.
+> If Scoop doesn't know where the bucket is located its repository location must be specified.
```
Comment (navarroaxel) [inline pages.zh/windows/scoop-bucket.md:4]: Did you forget to translate this? :grin:

File: pages.zh/windows/takeown.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# takeown
+
+> 取得文件或目录的所有权.
+> 更多信息: <https://docs.microsoft.com/windows-server/administration/windows-commands/takeown>.
+
+- 取得指定文件的所有权:
+
+`takeown /f {{path/to/file}}`
```
Comment (navarroaxel) [inline pages.zh/windows/takeown.md:8]: Could you translate the `path/to/file` to be easy to read for non-English speakers? the same for other commands here and `wsl`.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @ChungZH, thanks for your submission. I left some comments.

**Review by navarroaxel — APPROVED** [review]
LGTM, thanks @ChungZH

**Review by schneiderl — APPROVED** [review]
Don't know the language but looks good.

**Pull request merged**